data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/DecimalUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.scalar.db.dataloader.core.util;
+
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
+
+/** Utils for decimal handling */
+public class DecimalUtil {
+
+  private static final DecimalFormat DECIMAL_FORMAT = createFormatter();
```

Comment:
`DecimalFormat` isn't thread-safe:
https://www.baeldung.com/java-decimalformat#thread-safety

We should not share the same instance between threads.

Reply:
@brfrn169 san,

I had changed this based on an earlier comment. I have reverted this back.